src/context/input_reader.rs

```diff
@@ -153,6 +153,58 @@ impl InputReader<'_, '_> {
                 value.into()
             }
             Binding::None => false.into(),
+            Binding::AnyKey => {
+                if self.ignored(Binding::AnyKey) {
+                    return false.into();
+                }
+                let keyboard = self
+                    .keys
+                    .get_pressed()
+                    .filter(|_| self.action_sources.keyboard)
+                    .map(|key| Binding::Keyboard {
+                        key: *key,
+                        mod_keys: ModKeys::empty(),
+                    });
+                let mouse_buttons = self
+                    .mouse_buttons
+                    .get_pressed()
+                    .filter(|_| self.action_sources.mouse_buttons)
+                    .map(|button| Binding::MouseButton {
+                        button: *button,
+                        mod_keys: ModKeys::empty(),
+                    });
```

Comment:
I think you can use this here and in other places.
```suggestion
                    .map(Into::into);
```

Reply:
Trying to replace it with that gives me an error:
```
error[E0277]: the trait bound `binding::Binding: From<&bevy::prelude::KeyCode>` is not satisfied
160 |                   let keyboard = self
    |  ________________________________^
161 | |                     .keys
162 | |                     .get_pressed()
163 | |                     .filter(|_| self.action_sources.keyboard)
164 | |                     .map(Into::into);
    | |____________________________________^ the trait `From<&bevy::prelude::KeyCode>` is not implemented for `binding::Binding`
```